src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java

```diff
@@ -7,6 +7,9 @@
 
 public class MultiRangeParams {
 
+  private long[] timestamps;
```

Comment:
@sazzad16 please rename these fields to something with filter. e.g `timestampsFilter`